driver/src/test/scala/Common.scala

```diff
@@ -167,4 +166,45 @@ object Common {
 
     slowProxy.stop()
   }
+
+}
+
+trait CommonAuth {
+
+  import reactivemongo.api.AuthenticationMode
+
+  def driver: MongoDriver
+
+  def authMode: Option[AuthenticationMode] =
+    Option(System.getProperty("test.authMode")).flatMap {
+      case "cr"   => Some(CrAuthentication)
+      case "x509" => Some(X509Authentication)
+      case _      => None
+    }
+
+  def defaultAuthentications: Seq[Authenticate] =
+    ifX509(Seq(Authenticate("", certSubject, "")))(otherwise = Nil)
+
+  def certSubject: String =
+    Option(System.getProperty("test.clientCertSubject"))
+      .getOrElse(sys.error("Client cert subject required if X509 auth enabled"))
+
+  def increaseTimeoutIfX509(timeout: FiniteDuration): FiniteDuration =
```

Comment:
Why?

Reply:
Tests got very flaky until I've increased the timeout. Running individual classes works but running all it tests was flaky

Reply:
If the suite runs fine locally, this extra timeout should be removed for now until a cause for it could be clearly identified